packages/@webex/http-core/src/request/index.js

```diff
@@ -18,6 +18,10 @@ export default function request(options) {
 
   options.headers = options.headers || {};
 
+  // it will take timeout if passed from request options
+  // else will add default timeout of 5 secs(5000ms)
+  options.timeout = options.timeout || 5000;
```

Comment:
5s is probably too small. For me the request we do to https://webexapis.com/v1/meetingPreferences  always takes somewhere between 4s and 6s. Also the addMedia request that sends Roap offer takes sometimes over 15s when Linus reports 503 (for example if we send Turn Discovery Request and the server has the feature toggle for that disabled).
So, either this default should be higher, something like 30s or specific bigger timeouts need to be set for the 2 requests I've mentioned, but I don't know how many other "slow" requests we have.